Adafruit_BluefruitLE/bluez_dbus/device.py

```diff
@@ -22,22 +21,25 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-from builtins import map
-from builtins import str
-from past.builtins import map
+
 import threading
 import time
 import uuid
 
 import dbus
 
+from builtins import map
+from builtins import str
+from past.builtins import map
+
 from ..config import TIMEOUT_SEC
 from ..interfaces import Device
 from ..platform import get_provider
 
 from .adapter import _INTERFACE as _ADAPTER_INTERFACE
 from .gatt import BluezGattService, BluezGattCharacteristic, _SERVICE_INTERFACE, _CHARACTERISTIC_INTERFACE
 
+from __future__ import unicode_literals
```

Comment:
Looks good, just move `__future__` imports to the very top
E.g. in the case, it should be 
```
from __future__ import unicode_literals

import threading
import time
import uuid

import dbus

from builtins import map
from builtins import str
from past.builtins import map

from ..config import TIMEOUT_SEC
from ..interfaces import Device
from ..platform import get_provider

from .adapter import _INTERFACE as _ADAPTER_INTERFACE
from .gatt import BluezGattService, BluezGattCharacteristic, _SERVICE_INTERFACE, _CHARACTERISTIC_INTERFACE
```

Reply:
Ok. Thanks